tests/torchscript_example_test.py

```diff
@@ -0,0 +1,61 @@
+import torch
+import unittest
+import numpy as np
+
+class TestTorchScriptModel(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Load the TorchScript model
+        cls.model = torch.jit.load('notebooks/simple_model_scripted.pt')
```

Comment:
We like to avoid checking in the model binaries. Can you please recreate it here and use it for your test, and then remove the binary file?